kolibri/core/tasks/job.py

```diff
@@ -156,9 +171,40 @@ def __getstate__(self):
             "args",
             "kwargs",
             "func",
-            "result",
         ]
-        return {key: self.__dict__[key] for key in keys if key in self.__dict__}
+
+        if isinstance(self.exception, Exception):
+            self.exception = type(self.exception).__name__
```

Comment:
We shouldn't coerce the exception here - this is where the exception gets recorded into the ORM, so we should do the coersion there:

https://github.com/learningequality/kolibri/blob/6903afec0ca3ebdf65fbf0eb96ef5348ed9fc2d2/kolibri/core/tasks/storage.py#L243